src/Runner.php

```diff
@@ -26,10 +27,11 @@ class Runner
      */
     public function __construct(Container $container = null)
     {
+        Robo::setContainer($container);
         $commands_directory = __DIR__ . '/Commands';
         $top_namespace = 'Pantheon\Terminus\Commands';
         $this->commands = $this->getCommands(['path' => $commands_directory, 'namespace' => $top_namespace,]);
-        $this->runner = new RoboRunner(null, null, $container);
+        $this->runner = new RoboRunner();
```

Comment:
When I use that, it tells me there's no `setContainer` function in `Robo\Runner`.